content/docs/command-reference/exp/init.md

```diff
@@ -18,14 +18,13 @@ usage: dvc exp init [-h] [-q | -v] [--run] [--interactive] [-f]
 
 ## Description
 
-`dvc exp init` helps you get started with DVC Experiments quickly. It reduces
-boilerplate DVC procedures by creating a `dvc.yaml` file that assumes standard
-locations of your input data, <abbr>parameters</abbr>, source code, models,
-<abbr>metrics</abbr> and [plots](/doc/command-reference/plots). These locations
-can be customized through the [options](#options) below or via
-[configuration](/doc/command-reference/config#exp).
-
-Repository structure assumed by default:
+`dvc exp init` helps you get started with DVC Experiments quickly. It prepares
+an experiment in `dvc.yaml`, which assumes standard locations for the inputs
```

Comment:
```suggestion
`dvc exp init` helps you get started with DVC Experiments quickly. It prepares
the experiment structure in `dvc.yaml`, which assumes standard locations for the inputs
```

Feel free to reword, but I think "prepares an experiment" makes it sound like it's something you do for every experiment instead of a framework for all your experiments.

Reply:
Actually, I think it's hard to separate this from #3430. These look they will inevitably conflict.

Reply:
Overlap is OK, we can solve conflicts 🙂

Reply:
Hmmm wait yeah you're right. This is now included completely in #3430... Closing this PR then.